nix/drun.nix

```diff
@@ -18,29 +18,28 @@ pkgs:
       cargoLock = {
         lockFile = "${pkgs.sources.ic}/Cargo.lock";
         outputHashes = {
+          "build-info-0.0.27" = "sha256-SkwWwDNrTsntkNiCv6rsyTFGazhpRDnKtVzPpYLKF9U=";
           "derive_more-0.99.8-alpha.0" = "sha256-tEsfYC9oCAsDjinCsUDgRg3q6ruvayuA1lRmsEP9cys=";
-          "ic-btc-interface-0.1.0" = "sha256-VecEMFjoeiRi0VgJ9CeDoOzdyJbJNiZ5MBmiV1+b7As=";
+          "ic-btc-interface-0.1.0" = "sha256-JoVg1t62C2FIe0la1oQzidybLj1CyAQy80gkRh/MTn0=";
+          "ic-btc-test-utils-0.1.0" = "sha256-VecEMFjoeiRi0VgJ9CeDoOzdyJbJNiZ5MBmiV1+b7As=";
+          "icrc1-test-env-0.1.1" = "sha256-yWJF+KM8l65Nr0pwR9QeltkqbHDzOLNPVnLhf1mRukQ=";
           "jsonrpc-0.12.1" = "sha256-3FtdZlt2PqVDkE5iKWYIp1eiIELsaYlUPRSP2Xp8ejM=";
           "libssh2-sys-0.2.23" = "sha256-9Hb7CnPF+lxrVO1NAhS7EXcPVWZutJXr6UWxpptzk4U=";
-          "lmdb-rkv-0.14.99" = "sha256-dqxWgtNZ/qRukQLiDawjHRwdPgfzm/sVcWwudwX9rD8=";
-          "pprof-0.10.1" = "sha256-ioZ8AyFTUUev8MDZapto0yXC6G+dZzg+7ZNtTR87Rg4=";
+          "lmdb-rkv-0.14.99" = "sha256-5WcUzapkrc/s3wCBNCuUDhtbp17n67rTbm2rx0qtITg=";
         };
       };
 
       patchPhase = ''
         cd ../cargo-vendor-dir
         patch librocksdb-sys*/build.rs << EOF
-@@ -118,6 +118,10 @@
-         config.define("OS_MACOSX", Some("1"));
-         config.define("ROCKSDB_PLATFORM_POSIX", Some("1"));
-         config.define("ROCKSDB_LIB_IO_POSIX", Some("1"));
-+        if target.contains("aarch64") {
-+            config.define("isSSE42()", Some("0"));
-+            config.define("isPCLMULQDQ()", Some("0"));
-+        }
-     } else if target.contains("android") {
-         config.define("OS_ANDROID", Some("1"));
-         config.define("ROCKSDB_PLATFORM_POSIX", Some("1"));
+@@ -249,6 +249,9 @@ fn build_rocksdb() {
+         config.flag("-Wno-missing-field-initializers");
```

Comment:
So RocksDB now finally supports the M1?

Reply:
This change actually only supports MacOS with x64 emulation. 
We should also test it on M1 natively...

Reply:
Seems to work!

Reply:
Thanks a lot!